src/muya/lib/ui/quickInsert/config.js

```diff
@@ -47,108 +66,125 @@ export const quicInsertObj = {
   //   color: 'rgb(224, 54, 54)'
   // }],
   'basic block': [{
-    title: 'Text',
-    subTitle: 'Just start write plain text.',
+    title: 'Paragraph',
+    subTitle: 'Lorem Ipsum is simply dummy text',
     label: 'paragraph',
+    shortCut: `${COMMAND_KEY}0`,
```

Comment:
We cannot hard code the shortcuts because we load all shortcuts at runtime from the user configuration (`keybindings.getAccelerator('%id%')`). Please see [here](https://github.com/marktext/marktext/blob/8f73940e2e947863395dc51ce16ce3bc65caa58a/src/main/shortcutHandler.js#L18) and [here](https://github.com/marktext/marktext/blob/8f73940e2e947863395dc51ce16ce3bc65caa58a/src/main/shortcutHandler.js#L226).

Reply:
>We cannot hard code the shortcuts

👍 Yes, I'll update this in another PR, this PR focus the style and update the `package.json` file.